content/en/docs/tasks/configure-pod-container/security-context.md

```diff
@@ -474,6 +474,56 @@ can lose track of volume labels after restart. This deficiency has been fixed
 in Kubernetes 1.26.
 {{< /note >}}
 
+## Changing the permissions of your's '/proc'
+
+{{< feature-state for_k8s_version="v1.12" state="alpha" >}}
+
+By default, containers have multiple paths that are both
+[masked](https://github.com/opencontainers/runtime-spec/blob/f66aad47309/config-linux.md#masked-paths)
+and read only. The result of this is the container has these paths present
+inside the container's mount namespace, and they can function similarly to if
+the container was an isolated host, but the container process cannot write to
+them. A list of the paths are found in the [Kubernetes source
+code](https://github.com/kubernetes/kubernetes/blob/964529b227/pkg/securitycontext/util.go#L193-L211):
```

Comment:
Avoid linking to the source code; instead, if we want to put a list in, pop it inside a new (small) page within https://kubernetes.io/docs/reference/node/

Reply:
how do you feel about me dropping the link to the source code but not adding a reference? having it in 3 places seems more likely to skew eventually

Reply:
That's fine.